src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/CodeStyleHelpers.cs

```diff
@@ -62,24 +62,26 @@ public static bool TryParseBoolEditorConfigCodeStyleOption(string arg, CodeStyle
         public static bool TryGetCodeStyleValueAndOptionalNotification(
             string arg, NotificationOption2 defaultNotification, [NotNullWhen(true)] out string? value, [NotNullWhen(true)] out NotificationOption2 notification)
         {
-            var args = arg.Split(':');
-            Debug.Assert(args.Length > 0);
+            var firstColonIndex = arg.IndexOf(':');
 
             // We allow a single value to be provided without an explicit notification.
-            if (args.Length == 1)
+            if (firstColonIndex == -1)
```

Comment:
is it guaranteed to return -1?  or shoudl this be `<0` to be safe?

Reply:
The documentation indicates that it returns "The zero-based index position of value if that character is found, or -1 if it is not."

Reply:
wfm.